config/default/core.extension.yml

```diff
@@ -77,6 +77,7 @@ module:
   metatag: 0
   metatag_dc: 0
   metatag_open_graph: 0
+  migrate: 0
```

Comment:
Migrate module should not be enabled by default.

Reply:
Done

Suggestion for addition to pre-commit-hook:

### Check core.extensions for modules that should be disabled
pattern='migrate\|sample\|dblog'
if grep $pattern  config/default/core.extension.yml >> /dev/null; then
    echo "Illegal modules detected as enabled. Please disable listed above"
    grep $pattern config/default/core.extension.yml
    exit 1
fi
### End check core.extensions for modules that should be disabled